processors/numberlines.go

```diff
@@ -0,0 +1,50 @@
+package processors
+
+import (
+	"fmt"
+	"strings"
+)
+
+// / Prepends consecutive number to each line of input
+type LineNumberer struct{}
+
+func (p LineNumberer) Name() string {
+	return "number-lines"
+}
+
+func (p LineNumberer) Alias() []string {
+	return []string{"nl", "number-lines"}
+}
+
+func (p LineNumberer) Transform(data []byte, _ ...Flag) (string, error) {
+	var s = string(data[:])
+	var counter = 1
+	var result = ""
+	var err error
+	for line := range strings.Lines(s) {
+		if line == "\n" {
+			result += line
+		} else {
+			result += fmt.Sprintf("%d. %s", counter, line)
```

Comment:
We may consider to pad with space

Because

```
1. foo
…
9. foo
10. too
```

could be this

```
1.  foo
…
9.  foo
10. too
```

Or this

```
 1. foo
…
 9. foo
10. too
```

Reply:
Evil is in detail